README.md

```diff
@@ -0,0 +1,147 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="./assets/images/how-to-do-a-capstone-project.jpg" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>1st Capstone Project Of Microverse (Module One)</b></h3>
+
+</div>
+<br/>
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+ - [📺 Presentation](#presentation) 
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Capstone-project] <a name="about-project"></a>
+
+>🔊 This website is a web development course that was inspired from a conference website CC-Global-Summit of 2015 designed by [cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">Html</a></li>
+    <li><a href="https://developer.mozilla.org/en/docs/Web/CSS">Css</a></li>
+    <li><a href="https://www.javascripttutorial.net">JavaScript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Implement Linters**
+- **Implement Github flow**
+- **Implement mobile version**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo On Github Pages](https://aminederbal.github.io/capstone-project-1/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📺 Presentation <a name="presentation"></a>
+- [ScreenRec Presentation](https://screenrec.com/share/LaxkB6AzMU)
+
```

Comment:
- [ ] You have done a good job so far by touching all the required changes, but as observed by the previous reviewer, the video should contain a highlight about a piece of the code that you find interesting or proud of making. I would implore you to kindly record another video and make sure all the requirements are met before making another submission.
![image](https://user-images.githubusercontent.com/86200356/217469578-ce048a40-85de-4ba0-a1ca-c6cc24cb3b05.png)

Reply:
I did highlight the hamburger animation